core/commands/dag/dag.go

```diff
@@ -606,9 +606,16 @@ The output of blocks happens in strict DAG-traversal, first-seen, order.
 
 		// minimal user friendliness
 		if err != nil &&
-			!node.IsOnline &&
 			err == ipld.ErrNotFound {
```

Comment:
This seems redundant.

Reply:
Given that there could be more errors that have nothing to do with being offline (e.g. datastore fetching errors, context cancellations, etc.) and that this was already here doesn't seem like it hurts.